i18n/zh/docusaurus-plugin-content-docs/current/administrator/configuration/configure-controllers.md

```diff
@@ -119,7 +119,6 @@ For the Karmada control plane, we also provide the capability to clean up finish
 specifying the `.spec.ttlSecondsAfterFinished` field of a Job, which will be a relief for the control plane.
 
 More details please refer to:
-- [ttl after finished controller](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/#ttl-after-finished-controller)
```

Comment:
Can you add more details about this PR, and let us know why need to remove this?

Reply:
Deletion is easy and seems rather careless. You can change it to a new heading like:

```
- [Update TTL for finished Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/#updating-ttl-for-finished-jobs)
```